open-source-trumps-innersource.md

```diff
@@ -0,0 +1,42 @@
+## Title
+Open Source trumps InnerSource
+* ORIGINAL: The Invented Here bias or internally developed
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+* ORIGINAL: Developers tend to select open source components rather than internally developed components, which results in poorer quality components or also makes it difficult to sustain internal component development.
+* NEW: Different BLs are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.
```

Comment:
What does _BL_ stand for? Is it business line? Is this different from a _BU_ (business unit)?

Reply:
The BL acronym is spelled out in the Context section now. Resolving.